### DISCUSSION TITLE
Scheduler allPlayers offline

### DISCUSSION THREAD

**Discussion by JaydenJinx** [Zrips/CMI#3745] [Enhancement]
**Description of issue or feature request:**

I would like to send a command one time every hour for all online and offline players with a specific permission:

Example:

Fly:
Enabled: true
Repeat: true
Delay: 3600
Commands:
- allPlayers! perm:fly.hourly! flightcharge add [allPlayers] 100

This works, but only for online players. I can't find documentation on how to send a command for all offline players too.
Maybe just add an "allOfflinePlayers!" prefix?

**Comment by Zrips** [comment]
In general this could be quite expensive on server side. Processing online players is fine as you will have max of 200 or so in one server, while including offline players could include tens of thousands. Combine this with permission check and you have issue, as most permission plugins doesn't support permission check on offline player.
Will keep this as possible enchantment, but its hardly gonna be implemented.

**Comment by JaydenJinx** [comment]
Gotcha. Appreciate the response.
